src/server/cloak-server.js

```diff
@@ -37,6 +37,11 @@ module.exports = function(expressServer) {
             createRoom: (arg, user) => {
                 cloak.createRoom(arg);
                 fireRoomListReload();
+            },
+            joinRoom: (arg, user) => {
```

Comment:
I think that for better readability of this file these functions should be defined separately and assigned to the properties:
```
...
joinRoom: joinRoom
...

function joinRoom() {
    ...
}
```